service is_running error in Ubuntu 16.04 docker container

I'm using molecule with testinfra. I have two contaieners: ubuntu 16.04 and ubuntu 18.04. On new version the test work, but on the older I get:
```
$ molecule verify
(...)

    ============================= test session starts ==============================
    platform darwin -- Python 2.7.15, pytest-4.3.0, py-1.8.0, pluggy-0.9.0
    rootdir: /Users/tomasz/ansible-playbooks/plays/haproxy/molecule/default, inifile:
    plugins: testinfra-1.19.0
collected 4 items

    tests/test_default.py ...F                                               [100%]

    =================================== FAILURES ===================================
    ______________ test_service_is_running[ansible://ladon.amana.vpn] ______________

    host = <testinfra.host.Host object at 0x107235d50>

        def test_service_is_running(host):
            service = host.service('haproxy')

    >       assert service.is_running

    tests/test_default.py:26:
    _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    self = <service haproxy>

        @property
        def is_running(self):
            return self.run_expect(
    >           [0, 3], "systemctl is-active %s", self.name).rc == 0
    E       AssertionError: Unexpected exit code 1 for CommandResult(command=u'systemctl is-active haproxy', exit_status=1, stdout=u'', stderr=u'Failed to connect to bus: No such file or directory')
    E       assert 1 in [0, 3]
    E        +  where 1 = CommandResult(command=u'systemctl is-active haproxy', exit_status=1, stdout=u'', stderr=u'Failed to connect to bus: No such file or directory').rc

    /usr/local/lib/python2.7/site-packages/testinfra/modules/service.py:107: AssertionError
    ===================== 1 failed, 3 passed in 50.88 seconds ======================
```

--- a/testinfra/modules/service.py
+++ b/testinfra/modules/service.py
@@ -103,8 +103,12 @@ class SystemdService(SysvService):
 
     @property
     def is_running(self):
-        return self.run_expect(
-            [0, 3], "systemctl is-active %s", self.name).rc == 0
+        out = self.run_expect(
+            [0, 1, 3], "systemctl is-active %s", self.name)
+        if out.rc == 1:
+            # Failed to connect to bus: No such file or directory
+            return super(SystemdService, self).is_running
+        return out.rc == 0
 
     @property
     def is_enabled(self):

